psycop/common/model_training_v2/hyperparameter_suggester/test_optuna_hyperparameter_search.cfg

```diff
@@ -0,0 +1,48 @@
+[logger]
+@loggers = "terminal_logger"
+
+[project_info]
+experiment_path = "/."
+
+[trainer]
+@trainers = "split_trainer"
+uuid_col_name = "pred_time_uuid"
+training_outcome_col_name = "outcome"
+validation_outcome_col_name = "outcome"
+logger = {"@loggers":"terminal_logger"}
+metric = {"@metrics":"binary_auroc"}
+training_data = {"@data":"minimal_test_data"}
+validation_data = {"@data":"minimal_test_data"}
+
+[trainer.preprocessing_pipeline]
+@preprocessing = "baseline_preprocessing_pipeline"
+
+[trainer.preprocessing_pipeline.*.age_filter]
+@preprocessing = "age_filter"
+min_age = 4
+max_age = 99
+age_col_name = "pred_age"
+
+[trainer.task]
+@tasks = "binary_classification"
+
+[trainer.task.task_pipe]
+@task_pipelines = "binary_classification_pipeline"
+
+[trainer.task.task_pipe.sklearn_pipe]
+@task_pipelines = "pipe_constructor"
+
+[trainer.task.task_pipe.sklearn_pipe.*.logistic_regression]
+@suggesters = "logistic_regression_suggester"
+solvers = ["saga"]
+penalties = ["l1","l2","elasticnet"]
+
+[trainer.task.task_pipe.sklearn_pipe.*.logistic_regression.C]
+low = 0.00000001
+high = 10
+logarithmic = false
```

Comment:
After reading this, perhaps we should make the parameters a 3-item list? This would allow us to specify:

`C = [0.000001, 10, false]` 

Instead of a full section. Not necessarily in this PR, though 👍

Reply:
You were right - the functionality does not seem to be there already. Created an issue from your comment